pom.xml

```diff
@@ -13,10 +13,10 @@
         <easyb.version>1.4</easyb.version>
         <cobertura.version>2.6</cobertura.version>
         <!-- A workaround for a bug in PMD -->
-        <sourceJdk>1.7</sourceJdk>
-        <targetJdk>1.7</targetJdk>
+        <sourceJdk>1.8</sourceJdk>
+        <targetJdk>1.8</targetJdk>
         <github.account>wakaleo</github.account>
-        <thucydides.version>0.9.268</thucydides.version>
+        <thucydides.version>1.8.3</thucydides.version>
```

Comment:
It should be <thucydides.version>0.9.125</thucydides.version> cause when i changed in version 1.8.3 in my pom, mvn has error so to solve it, i change the version as 0.9.125